client/store/index.js

```diff
@@ -11,7 +11,9 @@ import nextExperiences from './nextExperiences'
 import selectedPlace from './selectedPlace'
 import checkinPlace from './checkinPlace'
 import currentLocation from './currentLocation'
+import checkUsername from './checkUsername'
```

Comment:
It looks like the checkUserName file default exports a function that puts a user error on the store.  Two comments on this:
1.  If so, the import should be named user error.  
2.  Can you combine the two user error store items into one?  If not, can you name them more clearly?

Reply:
it's not the same as user error. In the first place, whether the username is available or not, it's not necessarily an error. It's just a button to see if a username is available or not. the returned message from the backend is just a response like any old res.json,